gravatar-quickeditor/src/main/java/com/gravatar/quickeditor/ui/components/SelectableAvatar.kt

```diff
@@ -90,8 +91,9 @@ internal fun SelectableAvatar(
 
         if (moreOptionsPopupVisible) {
             AvatarMoreOptionsPickerPopup(
-                anchorAlignment = Alignment.CenterHorizontally,
+                anchorAlignment = Alignment.Start,
                 anchorBounds = popupAnchorBounds,
+                maxBounds = parentBounds,
```

Comment:
Completely unrelated to this PR, but just wanted to share that I'm having doubts about whether this Popup should be inside the `SelectableAvatar`. It feels like too much for such a small composable, and now with the extra params that have to be passed just for that, this makes it clearer I think. 

@hamorillo Thoughts on this?

Reply:
I have the same feeling. In fact, when I implemented the Popup, I had several doubts about whether it should be outside the SelecrtableAvatar. 

The benefit of having the Popup within the SelectableAvatar is that it's encapsulated in just one place. However, I feel now that we are adding a responsibility that Composable should be outside and managed by a parent composable. 

Said that, I would create an issue with the lowest priority for now.

Reply:
Done https://github.com/Automattic/Gravatar-SDK-Android/issues/457